hypothesis-python/tests/cover/test_type_lookup.py

```diff
@@ -118,6 +118,14 @@ def test_lookup_overrides_defaults(typ):
     assert st.from_type(typ).example() is not sentinel
 
 
+def test_lookup_registered_tuple():
+    sentinel = object()
+    typ = tuple[int]
```

Comment:
This will fail on Python 3.8, because builtin generics only became subscriptable in Python 3.9.  The solution is to move this test to `hypothesis-python/tests/cover/test_lookup_py39.py` 🙂